tests/page-objects/reports/reports.wdio.page.js

```diff
@@ -110,9 +115,77 @@ const reportsListDetails = async () => {
   return reportDetails;
 };
 
+const collapseSelection = async () => {
+  await (await itemSummary()).click();
+  expect(await (await reportBodyDetails()).isExisting()).to.be.false;
+};
+
+const deleteSelectedReports = async (savedUuids) => {
```

Comment:
I'm not sure I understand ... why is this function receiving an array of uuids and then casually making an assertion about one of them, when the function name is "deleteSelectedReports"? 
I think this function should not receive any parameters and not make any assertions about remaining reports. All assertions should be done separately, either in the test file itself or in another function.